tox.ini

```diff
@@ -0,0 +1,29 @@
+[tox]
+envlist = py{27,35},coverage
+
+[testenv]
+deps =
+    coveralls
+    ddt
+    django>=1.8,<1.9
```

Comment:
FYI, my changes that we needed to run on django 1.9 just got merged into the most recent version of markey, which is a dependency of some other thing we use that I don't recall, anyway point being you can probably start testing i18n-tools on django 1.9 now.